default/AI/AIstate.py

```diff
@@ -120,8 +127,28 @@ def __setstate__(self, state_dict):
         for odict_attrib in ['colonisablePlanetIDs', 'colonisableOutpostIDs']:
             if dict_attrib not in state_dict:
                 self.__dict__[odict_attrib] = odict()
+        if 'uuid' not in state_dict:
+            self.uuid = uuid4()
+        if 'turn_hashes' not in state_dict:
```

Comment:
did you change the attribute name in one line but not the other? 'turn_hases' vs. 'turn_uuids'